libvmaf/README.md

```diff
@@ -1,15 +1,60 @@
 # libvmaf
 
+## Prerequisites
+
+For building, you need the following:
+
+- [Meson](https://mesonbuild.com/) (0.47 or higher)
+- [Ninja](https://ninja-build.org/)
+
+Install these dependencies under Ubuntu with:
+
+```
+sudo apt update -qq && \
+sudo apt install python3 python3-pip python3-setuptools python3-wheel ninja-build
+pip3 install --user meson
```

Comment:
Would it be better to recommend installing ninja through pip too?

Reply:
I don't have any preference here. I chose the distro-packaged version since it "just worked".

Reply:
Me neither. If there's not an official "best practice", then I'm fine with either.